src/main/webapp/event-info-page/script.js

```diff
@@ -0,0 +1,11 @@
+function createMap() {
```

Comment:
We can discuss the Map function together later and have a consistent map display.